src/collections/Breadcrumb/BreadcrumbSection.js

```diff
@@ -0,0 +1,98 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import META from '../../utils/Meta'
+import { customPropTypes, useKeyOnly } from '../../utils/propUtils'
+
+/**
+ * A section sub-component for Breadcrumb component.
+ *
+ * @see Breadcrumb
```

Comment:
The `@see` tag will show a link to a related component on the doc site:

![image](https://cloud.githubusercontent.com/assets/5067638/16599493/648ffa84-42b7-11e6-9ada-bf51297c1e60.png)

Subcomponents however do not yet show up on the doc site.  Let's remove the `@see` tag here since it will not do anything.

***

Side note, we will likely expose subcomponents in the future, but it will be automated.  We can do this since the _meta property already defines the parent component.